cmd/jb/init.go

```diff
@@ -15,32 +15,38 @@
 package main
 
 import (
+	"encoding/json"
 	"io/ioutil"
 	"path/filepath"
 
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/jsonnet-bundler/jsonnet-bundler/pkg/jsonnetfile"
+	"github.com/jsonnet-bundler/jsonnet-bundler/spec"
 )
 
 func initCommand(dir string) int {
 	exists, err := jsonnetfile.Exists(jsonnetfile.File)
-	if err != nil {
-		kingpin.Errorf("Failed to check for jsonnetfile.json: %v", err)
-		return 1
-	}
+	kingpin.FatalIfError(err, "Failed to check for jsonnetfile.json")
 
 	if exists {
 		kingpin.Errorf("jsonnetfile.json already exists")
 		return 1
 	}
 
+	// default to go-style only for new setups
```

Comment:
I was under the assumption that we agreed on defaulting to having both go-style and legacy imports enabled. (maybe it's just this comment that needs to be adapted)

Reply:
Yeah, an old comment